.github/scripts/do-rtl-build.sh

```diff
@@ -59,6 +59,7 @@ else
 fi
 
 # choose what make dir to use
+
```

Comment:
Nit: Can you just revert this so it stays the same.

Reply:
will do